.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": false,
+  "singleQuote": true
+}
```

Comment:
This file should not be required, and it doesn't adhere to the WP JS Coding Standards

The `@wordpress/scripts` package includes [this Prettier config](https://github.com/WordPress/gutenberg/blob/master/packages/prettier-config/lib/index.js#L15-L26)

Reply:
👍